Source/WebCore/rendering/RenderImage.cpp

```diff
@@ -710,7 +732,15 @@ ImageDrawResult RenderImage::paintIntoRect(PaintInfo& paintInfo, const FloatRect
         settings().showDebugBorders() ? ShowDebugBackground::Yes : ShowDebugBackground::No
     };
 
-    auto drawResult = paintInfo.context().drawImage(*img, rect, options);
+    auto drawResult = ImageDrawResult::DidNothing;
+#if ENABLE(MULTI_REPRESENTATION_HEIC)
+    if (isMultiRepresentationHEIC())
+        drawResult = drawMultiRepresentationHEICIntoContext(paintInfo.context(), style().fontCascade().primaryFont(), *img, rect);
```

Comment:
Why is this function not taking the `options` as an argument? Does not it care about the `compositeOperation` and `blendMode`?

Reply:
We now use the `options`.